scripts/start-all.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+# Copyright (c) 2019 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+SCRIPT_PATH=$(cd "$(dirname "$0")";pwd)
+
+echo "Start Meta Service ..."
+
+if [ ! -f ${SCRIPT_PATH}"/metas" ]; then
+  echo "${SCRIPT_PATH}/metas not exist"
+  exit 0
+fi
+
+for host in `cat ${SCRIPT_PATH}/metas`
+do
+  echo "start $host"
+  ssh $host "/usr/local/scripts/nebula.service -c /usr/local/etc/nebula-metad.conf start metad >/dev/null 2>&1 &"
```

Comment:
Users need to configure the secret key for ssh manually, right?

Reply:
yeap , if you don't setting the SSH password less login you have to input the password.